how-to/register-with-home/client/src/home.ts

```diff
@@ -0,0 +1,246 @@
+import {
+  Home,
+  CLIProvider,
+  CLISearchListenerRequest,
+  CLIFilter,
+  CLISearchResult,
+  CLISearchListenerResponse,
+  CLISearchResponse,
+  CLIDispatchedSearchResult
+} from "@openfin/workspace";
+import { getSettings } from "./settings";
+import { launch } from "./launch";
+import { getApps } from "./apps";
+
+function getSearchFilters(tags: string[]): CLIFilter[] {
+  if (Array.isArray(tags)) {
+    let filters: CLIFilter[] = [];
+    let uniqueTags = [...new Set(tags.sort())];
+    let tagFilter: CLIFilter = {
+      title: "Tags",
+      type: "MultiSelect" as any,
```

Comment:
importing enums and using them as value still doesn't work?

Reply:
Just tested. The enums work now so updated their usage in store.ts and home.ts.

Reply:
Awesome!